packages/context-entries/lib/Keys.ts

```diff
@@ -60,6 +60,16 @@ export enum KeysInitSparql {
    *                 This is required for certain SPARQL operations such as NOW().
    */
   queryTimestamp = '@comunica/actor-init-sparql:queryTimestamp',
+  /**
+   * @range {functionNamedNode: RDF.NamedNode) => ((args: RDF.Term[]) => Promise<RDF.Term>) | null} A callback function
```

Comment:
```suggestion
   * @range {functionNamedNode: RDF.NamedNode) => ((args: RDF.Term[]) => Promise<RDF.Term>) | undefined} A callback function
```
A common TS convention is to never use `null`, but always use `undefined` instead.